pulp_ansible/app/tasks/collections.py

```diff
@@ -344,7 +344,7 @@ async def _get_collection_api(root):
                 api_data = parse_metadata(await downloader.run())
             except (json.decoder.JSONDecodeError, ClientResponseError):
                 # users can use "https://galaxy.ansible.com" as a url so also try {root}/api/
-                root = f"{root}/api/"
+                root = f"{root}/api/" if "/api/" not in root else f"{root}/"
```

Comment:
it was adding an additional `/api/`:
```
Oct 28 21:55:18 pulp3-source-fedora31.localhost.example.com rq[47027]: aiohttp.client_exceptions.ClientResponseError: 404, message='Not Found', url=URL('https://cloud.redhat.com/api/automation-hub/api/v3/collections/ansible/posix/')  
```
the right URL would be: https://cloud.redhat.com/api/automation-hub/v3/collections/ansible/posix/
AH needs the trailing /
https://cloud.redhat.com/api/automation-hub/